### DISCUSSION TITLE
Formatting of display results

### DISCUSSION THREAD

**Discussion by saohoa** [goldendict/goldendict#1409] [Q&A] [answered]
Hi, I've been using gdict for many years, and I seem to remember the formatting of word descriptions to be better (this was probably on the linux version, not android). I'm using gdict pro on android, and the mtbab Vietnamese dictionaries (anhviet109k.dict.dz, vietanh.dict.dz, vv30k.dict.dz). The output of individual words is very poorly formatted and comes out like this:

 record
@record /'rekɔ:d/ * danh từ - (pháp lý) hồ sơ =to be on record+ được ghi vào hồ sơ; có thực (vì đã được ghi vào hồ sơ) =it is on record that...+ trong sử có ghi chép rằng... - biên bản - sự ghi chép - (số nhiều) văn thư; (thương nghiệp) sổ sách - di tích, đài, bia, vật kỷ niệm - lý lịch =to have a clean record+ có lý lịch trong sạch - (thể dục,thể thao) kỷ lục =to break (beat) a record+ phá kỷ lục =to achieve a record+ lập (đạt) một kỷ lục mới =to hold a record+ giữ một kỷ lục =world record+ kỷ lục thế giới - đĩa hát, đĩa ghi âm - (định ngữ) cao nhất, kỷ lục =a record output+ sản lượng kỷ lục =at record speed+ với một tốc độ cao nhất !to bear record to something - chứng thực (xác nhận) việc gì !to keep to the record - đi đúng vào vấn đề gì !off the record - không chính thức !to travel out of the record - đi ra ngoài đề[ri'kɔ:d] * ngoại động từ - ghi, ghi chép =to record the proceeding of an assembly+ ghi biên bản hội nghị - thu, ghi (vào đĩa hát, máy ghi âm) - chỉ =the thermometer records 32 o+ cái đo nhiệt chỉ 32o - (thơ ca) hót khẽ (chim) !to record one's vote - bầu, bỏ phiếu

record
@record - sự ghi - continous r. (máy tính) sự ghi liên tục

.....
Is there any way to format this nicely so it's nit all jumbled up? There's not many (any) alternatives so any help would be greatly appreciated. Thank you.

**Comment by vedgy** [comment, marked as answer]
GoldenDict for Android is a separate application, which is not maintained in this repository. You'd have to ask via a different channel. There must be a way for a customer to report issues to the software vendor.
